content/_data/upgrades/2-204-1.adoc

```diff
@@ -0,0 +1,71 @@
+==== Password fields in forms how hidden by default
+
+Password fields have been redesigned to prevent password auto-fill in password fields that are not theh initial login form.
```

Comment:
```suggestion
Password fields have been redesigned to prevent password auto-fill that are not in the initial login form.
```

Reply:
Done in 47c399f6545dae071c9723ce367c88a087119a19